apps/src/reportAbuse.js

```diff
@@ -0,0 +1,27 @@
+import cookies from 'js-cookie';
+import _ from 'lodash';
+
+/**
+ * Determines if the current user has already submitted an Abuse report for this project.
+ * @returns {boolean} true if the user has reported the given project for abuse.
+ */
+export const userAlreadyReportedAbuse = function(channelId) {
+  return (
+    cookies.get('reported_abuse') &&
+    _.includes(JSON.parse(cookies.get('reported_abuse')), channelId)
```

Comment:
as a follow-up work item, it'd be nice to see if we can get rid of the lodash dependency here and just use [`Array.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
created https://codedotorg.atlassian.net/browse/FND-1176